pkgs/by-name/br/brogue/package.nix

```diff
@@ -14,9 +14,21 @@ stdenv.mkDerivation (finalAttrs: {
   version = "1.7.5";
 
   src = fetchurl {
-    url = "https://sites.google.com/site/broguegame/brogue-${finalAttrs.version}-linux-amd64.tbz2";
-    sha256 = "0i042zb3axjf0cpgpdh8hvfn66dbfizidyvw0iymjk2n760z2kx7";
+    url = "https://drive.google.com/uc?export=download&id=1ED_2nPubP-P0e_PHKYVzZF42M1Y9pUb4";
+    hash = "sha256-p0/xgTlWTFl9BHz7Fn90qxlj3YYItvsuA052NdYXBEQ=";
   };
+
+  # Fix the "do not know how to unpack source archive" error
+  unpackPhase = ''
+    runHook preUnpack
+
+    tar -xf $src
+    sourceRoot=$(echo brogue-*)
+    chmod -R u+w $sourceRoot
+
+    runHook postUnpack
+  '';
+
```

Comment:
You can use [`unpackCmd`](https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-unpackCmd). Better yet, if you add the extension to the fetcher by adding `name = "brogue.tar.gz"` to `fetchurl` (or `extension = "tar.gz"` to fetchzip) automatic unpacking will work without having to specify the command

Reply:
always discovering new tricks!